themes/vpge/lib/scss/components/grad-grow-filter/_grad-grow-filter.scss

```diff
@@ -192,6 +193,37 @@
                 padding: 0.2rem 1.7rem;
                 font-weight: 700;
                 width: fit-content;
+            }
+
+              // Career Development
+              [data-id="1196"] {
+                color: $vpge-compentency-color-purple;
+                border: .2rem solid $vpge-compentency-color-purple;
+              }
+              // Communication
+              [data-id="1111"] {
+                color: $vpge-compentency-color-green;
+                border: .2rem solid $vpge-compentency-color-green;
```

Comment:
The border needs to be `0.2` instead of `.2`

Reply:
Done